applications/luci-app-upnp/root/usr/share/rpcd/acl.d/luci-app-upnp.json

```diff
@@ -0,0 +1,12 @@
+{
+	"luci-app-ddns": {
+		"description": "Grant access to upnp procedures",
+		"read": {
+			"ubus": {
+				"luci.upnp": [ "delete_rule", "get_status" ],
```

Comment:
Please move the `delete_rule` action into the `write` scope since it is a function with side effects. We also require `write` permissions on `upnpd` uci. End result should look like this:

```
{
	"luci-app-ddns": {
		"description": "Grant access to upnp procedures",
		"read": {
			"ubus": {
				"luci.upnp": [ "get_status" ],
				"luci": [ "setInitAction" ]
			}
		},
		"write": {
			"ubus": {
				"luci.upnp": [ "delete_rule" ]
			},
			"uci": [ "upnpd" ]
		}
	}
}
```